QUEST_JOBSTEP.tsv

```diff
@@ -61,36 +61,36 @@ QUEST_JOBSTEP_20150317_000060	Oh, this. I saw it somewhere...hmm...looks familia
-QUEST_JOBSTEP_20150317_000065	I used to be a thief, but I came to realize certain things and I stopped my life of crime. {nl}But there is a little load left on my mind...and I will resolve it now {nl}
+QUEST_JOBSTEP_20150317_000064	$Thank you, in the name of the Goddess.{nl}I promise the donations will be used for those injured by monsters. {nl}
+QUEST_JOBSTEP_20150317_000065	$I used to be a thief, but I came to realize certain things and I stopped my life of crime. {nl}But there is a little load left on my mind...and I will resolve it now. {nl}
 QUEST_JOBSTEP_20150317_000066	Anyway, you did a good job. {nl}Let's cooperate from now on, teaching and learning from each other. 
 QUEST_JOBSTEP_20150317_000067	I'm looking for a book. Four books by Lydia Schaffen. {nl}If you help me find the books, I will teach you my skills. 
-QUEST_JOBSTEP_20150317_000068	The book used to be in the Miner's town but it was lost when the Vubbes raided. {nl} Try going to the Vubbe Outpost.
+QUEST_JOBSTEP_20150317_000068	$The book used to be in the Miners' town, but it was lost when the Vubbes raided. {nl} Try going to the Vubbe Outpost.
 QUEST_JOBSTEP_20150317_000069	It's this book. You will need this book if you dream to be the best Archer. {nl}I will make you a copy if you can gather all the books. 
-QUEST_JOBSTEP_20150317_000070	Quarrel Shooter Master holds the next book. {nl}He was looking for Merog leather earlier. Maybe you can try a deal with him using that.
+QUEST_JOBSTEP_20150317_000070	$The Quarrel Shooter Master holds the next book. {nl}He was looking for Merog leather earlier. Maybe you can try making a deal with him using that.
 QUEST_JOBSTEP_20150317_000071	Do you have the Merog leather yet from the Hunter Master?
 QUEST_JOBSTEP_20150317_000072	This is of good quality. The Hunter Master will be pleased. {nl}You can take the book. {nl}
-QUEST_JOBSTEP_20150317_000073	Merogs are in the Fallen Worry Forest, and the Quarrel Shooter Master is in Klaipeda. {nl}Someone might get ahead of you so better hurry.
-QUEST_JOBSTEP_20150317_000074	The Archer Master in Klaipeda has the next book. {nl}He's not a mean person so he will give you the book if you ask nicely. {nl}
-QUEST_JOBSTEP_20150317_000075	He is helping with the repair of the damaged city wall. {nl}I heard there are rocks delivered to the Miner's Village, so bringing him that rock can be a way to ask. 
+QUEST_JOBSTEP_20150317_000073	$Merogs are in the Fallen Worry Forest, and the Quarrel Shooter Master is in Klaipeda. {nl}Someone might get ahead of you, so better hurry.
+QUEST_JOBSTEP_20150317_000074	$The Archer Master in Klaipeda has the next book. {nl}He's not a mean person, so he will give you the book if you ask nicely. {nl}
+QUEST_JOBSTEP_20150317_000075	$He is helping with the repair of the damaged city wall. {nl}I heard rocks were delivered to the Miners' Village, and bringing him a rock can be a way to ask. 
```

Comment:
I'm not sure if the name of the quest item is "rock," "boulder" or "stone." I also don't know if it's supposed to be plural. I switched everything to "rock" because that was the most common, even though it ~does~ sound kind of odd (who takes only one rock?) It can easily be changed in the future if that's not what it's supposed to be.